TwilioVerifyDemo/TwilioVerifyDemo/CreateFactor/Presenter/CreateFactorPresenter.swift

```diff
@@ -97,9 +99,10 @@ extension CreateFactorPresenter: CreateFactorPresentable {
         withFactorName: factorName,
         devicePushToken: devicePushToken,
         metadata: metadata,
+        allowIphoneMigration: allowIphoneMigration,
         success: { factor in
             
-        strongSelf.verify(factor, success: { _ in
+        strongSelf.verify(factor, allowIphoneMigration: allowIphoneMigration, success: { _ in
```

Comment:
Can we remove `allowIphoneMigration` from verify method and use the stored value for the factor?

Reply:
You can find an example of adding a new property to the factor model here: https://github.com/twilio/twilio-verify-ios/commit/4b6e296ddf0120e6b04f0139f1f43c0ff4f0c724, the default value should be false (this will apply for old factors)